python/lsst/ctrl/mpexec/cli/opt/optionGroups.py

```diff
@@ -0,0 +1,123 @@
+# This file is part of ctrl_mpexec.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+__all__ = ("butler_options", "execution_options", "meta_info_options", "pipeline_build_options",
+           "qgraph_options")
+
+
+from functools import partial
+
+from lsst.daf.butler.cli.utils import option_section, split_kv, unwrap
+import lsst.obs.base.cli.opt as obsBaseOpts
+import lsst.daf.butler.cli.opt as dafButlerOpts
+from . import options as ctrlMpExecOpts
+
+instrumentOptionHelp = ("Add an instrument which will be used to load config overrides when defining a "
+                        "pipeline. This must be the fully qualified class name.")
+
+
+class OptionGroup():
```

Comment:
Did you want to add base class inside parentheses?

Reply:
No, but I guess I can remove the parens?

Reply:
👍